### DISCUSSION TITLE
ایجاد تغییرات زیر برای بهبود رابط کاربری

### DISCUSSION THREAD

**Discussion by nsa14** [marzneshin/marzneshin#442] [Ideas]
اولا دمتون گرم
مرسی مرسی بابت تلاشتون
🌹🌹🌹

چون تجربه مرزبان رو دارم این تغییرات میتونه خیلی عالی باشه برا کاربران که خدمتتون عرض  میشه:

۱.
تو بخش theme dark
 رنگ مشکی اصلا مناسب نیستتتتت و چشمو اذیت میکنه
تغییر بدین به ی رنگ سورمه ایی مانند ک‌استانداردش هم همینه

 ۲.
یکم modalBox ها کند باز میشه
وقتی کلیک میشه تاخیرش در خدیه ک فکر میکنیم گیر کرده و صفحه رو رفرش میکنیم
سرعتش بهبود پیداکنه عالیه

۳.
نسخه pwa ک نداشت بخام تست کنم 

۴.
******* 
نمایش تعداد کاربر متصل به هر کانفیگ -مورد خیلی مهم و کاربردیه-صرفا فقط نمایش باشه تو مرحله بعدش اگر شد محدودبت بشه گذاشت

۵.
منوی پایین تو موبایل خیلی بدجاستتتت
بیاد تو منوی همبرگری موبایل بالای 
language
بنظرم هیلی بهتر میشه

**Comment by nsa14** [comment]
عدد روز مانده از تاریخ انقضا رو زیرش بنویسه بهتر میشه مثلا ۱۲ روز تا انقضا

تو باکس انتخابکر تاریخ وقتی تاریخی رو انتخاب کردیم تعداد روزشو پایین بنویسه مثلا ۱ ماه و ۱۰ روز

تو نمایش یوزرها 
قسمت 
Rows per page
عددش ذخیره بشه

تو نمایش موبایل
گزینه جستجو بالا کاربرد حیاتی نداره حذف بشه و بجاش منو  در دسترس تر باشه

تو نمایش موبایل منوی اصلی پایینه که بشدت بدجاست

**Comment by Krr0ptioN** [comment]
1. فعلا تصمیم به تغییر تم نیستیم. اولویت نیست و بدون نمونه هم نمیشه تصمیم خاصی درمورد توصیتون انجام بدیم. سر و روی داشبورد تغییرات زیادی خواهد کرد، اما در حال حاضر اولیت نیست.

2. #462
3. در نظر گرفته میشه

4. 
- #463
- #464

5. به چه دلیل تو جای بدیه؟

**Reply by nsa14** [reply]
مورد 5:

بخاطر اینکه اسکرول میخوره میره پایین صفحه و بعضا دیده نمیشه تو صفحه نمایش موبایل

اگر پایین میزارین ،منطقی اینه که فیکسش کنید که بچسبه به پایین و همیشه در دسترس باشه

**Reply by Krr0ptioN** [reply]
خب این باگه، ایشیو کنید با مراحل باز سازیش و سایز صفحهتون.
